### PR TITLE
[AUT-3641] CORE: Free form choice is not resizable in Graphic Interactions during Item Authoring

### PR DESCRIPTION
**Pull request by KirylHatalski**
Ticket: [AUT-3641](https://oat-sa.atlassian.net/browse/AUT-3641)

## What's Changed

- Fixed error on free form points drugging, error was related missing scope after lodash migration

## Demo
image

## How to test
- Launch the authoring with HotSpot interaction
- Create free form zone
- Edit it by drugging any point

[Env](http://test-kiril.playground.kitchen.it.taocloud.org:40351/) admin/admin

[AUT-3641]: https://oat-sa.atlassian.net/browse/AUT-3641?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: views/js/qtiCreator/widgets/interactions/helpers/shapeEditor.js [hunk 1]
```diff
@@ -192,13 +192,15 @@ define([
...
+                    }
+
                     if(self.shape.type === 'path'){
-                       _.forEach(self.shape.attr('path'), function(point, index){
-                           if(point.length === 3 && point[1] === this.attr('cx') && point[2] === this.attr('cy')){
-                                this.pointIndex = index;
-                                return false;
-                           }
-                       }, handler);
+                        self.shape.attr('path').forEach(setPointIndex.bind(handler));
```
Comment (jsconan) [inline views/js/qtiCreator/widgets/interactions/helpers/shapeEditor.js:203]: **suggestion:** No need to bind the function. `array.forEach()` already have auto binding, see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach#syntax
```suggestion
                        self.shape.attr('path').forEach(setPointIndex, handler);
```

**Review by jsconan — COMMENTED** [review]
It works well!

Just one remark as `.bind()` is not required.

File: views/js/qtiCreator/widgets/interactions/helpers/shapeEditor.js (hunk 1 above)
Comment (KirylHatalski) [reply views/js/qtiCreator/widgets/interactions/helpers/shapeEditor.js]: done

**Review by jsconan — APPROVED** [review]
**Review Checklist**
- [ ] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code respects code style rules
- [x] New code respects best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`
- [x] Commits are following [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] Commits messages are meaningful
- [x] Commits are atomic

**Review by Karol-Stelmaczonek — APPROVED** [review]

**Pull request merged**